PoshRedisViewer/PoshRedisViewer/UIUtil.fs

```diff
@@ -270,7 +270,20 @@ module ListView =
             | _ -> ()
         )
         listView
+    let addDetailedViewOnEnterKey (listView: ListView) =
```

Comment:
Please indent the code from lines 275 to 286 to the right and add a "return" listView at the end. Also, please add new lines before and after this function. You can use code snipped below as reference:
```fsharp
let addDetailedViewOnEnterKey (listView: ListView) =
    listView.add_KeyDown (fun keyDownEvent ->
        let validIdx = listView.Source <> null && listView.SelectedItem >= 0 && listView.SelectedItem < listView.Source.Count

        if validIdx then
            let selectedValue = listView.Source.ToList()[listView.SelectedItem]

            match keyDownEvent.KeyEvent.Key with
            | Key.Enter ->
                MessageBox.Query("Value",selectedValue.ToString(),"Ok")
                |> ignore
            | _ -> ()
            keyDownEvent.Handled <- true
    )
    listView // in F# everything is epression and there is no explicit `return` statement. The last expression provided will be returned automatically. Here it's better to return listView so the calls be chained via pipe operator.
```

Reply:
Got it, think that should be it fixed now in [7b5d25f](https://github.com/En3Tho/PoshRedisViewer/pull/11/commits/7b5d25f75c9e7782577af27999b296690da44b5c)